### PR TITLE
Add Crossbeam AtomicCell<*64> Soundness #1203

### LINKED ISSUE #1203: Crossbeam AtomicCell<*64> Soundness
crossbeam released a security update for 0.8.7. It would be good to have this in rustsec. 

https://github.com/crossbeam-rs/crossbeam/security/advisories/GHSA-qc84-gqf4-9926

### PR DESCRIPTION
**Pull request by pinkforest**
Closes #1203

### CODE REVIEW

File: crates/crossbeam-utils/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "crossbeam-utils"
+date = "2022-02-05"
+informational = "unsound"
+categories = ["memory-corruption"]
+url = "https://github.com/crossbeam-rs/crossbeam/pull/781"
+aliases = ["GHSA-qc84-gqf4-9926"]
```
Comment (amousset) [inline crates/crossbeam-utils/RUSTSEC-0000-0000.md:9]: it seems `CVE-2022-23639` is also an alias for this one

File: crates/crossbeam-utils/RUSTSEC-0000-0000.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+[versions]
+patched = [">= 0.8.7"]
+```
+
+# Unsoundness of AtomicCell<*64> arithmetics on 32-bit targets that support Atomic*64
+
+## Impact
+
+The affected version of this crate incorrectly assumed that the alignment of {i,u}64 was always the same as Atomic{I,U}64.
```
Comment (amousset) [inline crates/crossbeam-utils/RUSTSEC-0000-0000.md:19]:
```suggestion
The affected versions of this crate incorrectly assumed that the alignment of {i,u}64 was always the same as Atomic{I,U}64.
```

**Review by amousset — APPROVED** [review]
Apart from the small nitpick, lgtm

File: crates/crossbeam-utils/RUSTSEC-0000-0000.md (hunk 1 above)
Comment (pinkforest) [reply crates/crossbeam-utils/RUSTSEC-0000-0000.md]: Oh silly me I even had found it before :-) Thanks for picking it up

**Pull request merged**